components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1040,7 +1048,7 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
          vap_liq_exchange(its:ite,kts:kte),& ! OUT sun of vap-liq phase change tendencies
          vap_ice_exchange(its:ite,kts:kte),& ! OUT sum of vap-ice phase change tendencies
          vap_cld_exchange(its:ite,kts:kte),& ! OUT sum of vap-cld phase change tendencies
-         col_location(its:ite,:3)          & ! IN column locations 
+         col_location(its:ite,:3),         & ! IN column locations
```

Comment:
@PeterCaldwell 
Please erase the last comma, the code doesn't compile.

Reply:
Ok, fixed. Sorry about that - I'm going to get my programming license revoked if I keep making sloppy mistakes like this! Please try again.